tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3329,4 +3329,57 @@ public function userDownloadsTheSpaceUsingTheWebdavApi(string $user, string $spa
 			)
 		);
 	}
+
+	/**
+	 * @When user :user gets the content of file :fileName with URL :destinationUrl using WebDav API
+	 * @When user :user gets the content of file :fileName inside space :spaceName with URL :destinationUrl using WebDav API
+	 * @When user :user gets the content of file :fileName shared by user :sharer with URL :destinationUrl using WebDav API
+	 * @When user :user gets the content of file :fileName inside space :spaceName shared by user :sharer with URL :destinationUrl using WebDav API
+	 *
+	 * @param string $user
+	 * @param string $fileName
+	 * @param string $destinationUrl
+	 * @param string|null $spaceName
+	 * @param string|null $sharer
+	 *
+	 * @return void
+	 */
+	public function userGetsTheContentOfFileUsingWebDavAPI(
+		string $user,
+		string $fileName,
+		string $destinationUrl,
+		?string $spaceName = null,
+		?string $sharer = null
+	): void {
+		$user = $sharer ?? $user;
+		$user = $this->featureContext->getActualUsername($user);
+
+		if ($spaceName === null) {
+			$fileId = $this->featureContext->getFileIdForPath($user, $fileName);
+		} else {
+			$fileId = $this->getFileId($user, $spaceName, $fileName);
+		}
+
+		$fullUrl = $this->featureContext->getBaseUrl() . $destinationUrl . $fileId;
+		$this->featureContext->setResponse(
+			HttpRequestHelper::sendRequest(
+				$fullUrl,
+				'',
+				'GET',
+				$user,
+				$this->featureContext->getPasswordForUser($user)
+			)
+		);
+	}
+
+	/**
+	 * @Then the content of the file should be :expectedContent
+	 *
+	 * @param string $expectedContent
+	 *
+	 * @return void
+	 */
+	public function theContentOfTheFileShouldBe(string $expectedContent): void {
+		$this->featureContext->checkDownloadedContentMatches($expectedContent);
+	}
```

Comment:
we already have
https://github.com/owncloud/ocis/blob/8f70e63c81148a2d85c170a78b8449e7e05ca6db/tests/acceptance/features/bootstrap/WebDav.php#L1043-L1051

Why not use this? or if you need this exact wording add the definition here ^ instead of creating a new one